docs/guides/memcached/update-version/memcached.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Updating Memcached Database
+menu:
+  docs_{{ .version }}:
+    identifier: mc-update-version
+    name: Memcached
```

Comment:
Don't name this section Memcached. See RabbitMQ docs for better reference.

Do this for other sections too.